src/udiskslinuxfilesystem.c

```diff
@@ -2276,6 +2276,7 @@ handle_resize (UDisksFilesystem      *filesystem,
    */
   udisks_linux_block_object_trigger_uevent (UDISKS_LINUX_BLOCK_OBJECT (object));
 
+  udisks_filesystem_set_size (UDISKS_FILESYSTEM (filesystem), get_filesystem_size (UDISKS_LINUX_BLOCK_OBJECT (object)));
```

Comment:
I think the explicit retype `UDISKS_FILESYSTEM()` is not necessary here as it's the same data type.

Reply:
Ah, yes, different case than above. Thanks!